test/test_torch.py

```diff
@@ -15200,6 +15200,27 @@ def test_float_to_int_undefined_conversion(self, device, dtype):
         t = torch.tensor((-3.40282e+38, 3.40282e+38), device=device, dtype=torch.float)
         self.assertEqual(t.to(dtype).dtype, dtype)
 
+    # Note: CUDA will fail this test on most dtypes, often dramatically.
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    @onlyCPU
+    @dtypes(torch.bool, torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64)
+    def test_float_to_int_conversion_precision(self, device, dtype):
+        t = torch.tensor((-3.40282e+38, 3.40282e+38), device=device, dtype=torch.float)
+        a = np.array((-3.40282e+38, 3.40282e+38), dtype=np.float32)
```

Comment:
`np.finfo(np.float32).max`? np.finfo is numpy for std::numeric_limits.

Reply:
Sure.